### DISCUSSION TITLE
Using IKVM nuget on Windows 11 arm64

### DISCUSSION THREAD

**Discussion by ghost** [ikvmnet/ikvm#345] [Q&A]
Hi, I'm currently evaluating Windows 11 arm64 as build environment. When building a project that used to work on x64, I run into problems.

First at all: Is the target win10-arm64 fully supported?

The error I get is:

MSB4044	The "IkvmGetReferenceAssemblies" task was not given a value for the required parameter "ToolPath".

....\.nuget\packages\ikvm.msbuild\8.5.0\buildTransitive\netstandard2.0\IKVM.MSBuild.Tasks.targets

Regards
Wolfgang

**Comment by wasabii** [comment]
Nope. We can't yet produce executables for ARM64 platforms. And thus, the build and runtime tools can't be created.

**Comment by wasabii** [comment]
#197
